README.md

```diff
@@ -68,6 +68,7 @@ Please note that all configuration properties are optional.
 | `includeFailureMsg` | `BOOLEAN` | If this setting is set to true, this will output the detailed failure message for each failed test. | `false`
 | `includeConsoleLog` | `BOOLEAN` | If set to true, this will output all triggered console logs for each test suite. | `false`
 | `styleOverridePath` | `STRING` | The path to a file containing CSS styles that should override the default styling.* | `null`
+| `shouldUseCssFile` | `BOOLEAN` | If set to true, the CSS styles will use a .css file | `false`
```

Comment:
To make it a bit clearer about its purpose to the users:
*"If set to true, the CSS styles will link in the current theme's .css file instead of inlining its content on the page"*